### PR TITLE
Add base helm runtime func

### PR DESCRIPTION
**Pull request by chilianyi**

### CODE REVIEW

File: pkg/cmd/runtime/plugins/helm/helm_runtime.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	if err != nil {
+		return
+	}
+	return
+}
+
+func (p *HelmRuntime) setupTillerConnection(client kubernetes.Interface, restClientConfig *rest.Config, namespace string) (*kube.Tunnel, error) {
+	tunnel, err := portforwarder.New(namespace, client, restClientConfig)
+	if err != nil {
+		return nil, fmt.Errorf("Could not get a connection to tiller: %s\nPlease ensure you have run `helm init`", err)
```
Comment (chai2010) [inline pkg/cmd/runtime/plugins/helm/helm_runtime.go]: `%s` is for string, err should use `%v`

File: pkg/cmd/runtime/plugins/helm/helm_runtime.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+		return nil, err
+	}
+
+	return helm.NewClient(helm.Host(fmt.Sprintf("localhost:%d", tunnel.Local))), nil
+}
+
+func (p *HelmRuntime) getHelmClient() (helmClient *helm.Client, err error) {
+	client, clientConfig, err := p.getKubeClient()
+	if err != nil {
+		return nil, fmt.Errorf("Could not get a kube client: %s", err)
```
Comment (chai2010) [inline pkg/cmd/runtime/plugins/helm/helm_runtime.go]: ditto

File: pkg/cmd/runtime/plugins/helm/helm_runtime_test.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+)
+
+func TestHelmRuntime(t *testing.T) {
+	runtime := HelmRuntime{}
+
+	appConf := "~/.helm/cache/archive/zookeeper-0.4.2.tgz"
+
+	_, err := os.Stat(strings.Replace(appConf, "~/", os.Getenv("HOME")+"/", 1))
+	if err != nil {
+		fmt.Printf("Helm runtime test skipped because no [%s] %s", appConf, err)
```
Comment (chai2010) [inline pkg/cmd/runtime/plugins/helm/helm_runtime_test.go]: ditto

Comment (chilianyi) [reply pkg/cmd/runtime/plugins/helm/helm_runtime_test.go]: Ok, fixed

**Pull request merged**